src/cms-driver.ts

```diff
@@ -5,6 +5,7 @@ import * as api from './api';
 import {CacheEntryProperties, ExecutableTarget, RichTarget} from './api';
 import * as cache from './cache';
 import * as cms from './cms-client';
+import cfg from './config';
```

Comment:
Can you change `cfg` -> `config`, just to match the rest of the project.

Reply:
Then I will have to change every local varname with the `config` identifier. There are a couple of them in the `cms-driver.ts` file.
@vector-of-bool Should I do that?

Reply:
Go for it.

Reply:
Done.